index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <title>Document</title>
+</head>
+<body>
+    <div class="container">
+        <div class="toolbar">
+            <a class="toolbar_items logo_placeholder">
+                <h1 class="logo">My Logo</h1>
+            </a>
+            <a href="#" class="toolbar_items menu_icons">
+                <div class="one"></div>
+                <div class="two"></div>
+                <div class="three"></div>
+            </a>
+        </div>
+        <div class="headline">
+            <h2 class="Iam"> I am Wahidullah <br> nice to see you</h2>
+            <p class="InfoAbout">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <a href="#" class="letsConnect">Let's Connect</a>
```

Comment:
- [ ] I have realized that you did not use one CSS property useful for links that remove the underline decoration on the anchor `text-decoration: none;`, and you should do the same even for the social media links as well because they all present the underline.
- [ ] I can see that your design does not look like the original yet, let's see the difference below:

1. Your Toolbar
![image](https://user-images.githubusercontent.com/55465133/129406807-b984ea73-31e5-43e3-92ad-65adda3a8ab1.png)

1. The original Toolbar
![image](https://user-images.githubusercontent.com/55465133/129406891-eecbf40d-56b8-49ad-ac3e-8be80d1210de.png)

-> The difference here is the `font-family, space between elements`.

2. Your Main title and paragraph and connect button
![image](https://user-images.githubusercontent.com/55465133/129407186-717f9134-d7a5-46e0-be7a-7d5c32c36fad.png)

2. The original Title and paragraph and connect button
![image](https://user-images.githubusercontent.com/55465133/129407249-694ddc24-9843-430f-9d6b-2d93e7190a9a.png)

-> The difference here is the `font-size, color, line-height, text-decoration text-transform(uppercase)`

3. Your icon Links
![image](https://user-images.githubusercontent.com/55465133/129407538-cb6d44b1-d493-4da4-9b2d-51cf8e2ae5f0.png)

3. Original icon links
![image](https://user-images.githubusercontent.com/55465133/129407585-f1aa8982-655b-4457-9bfd-36d94eb97546.png)

-> The difference here is the `font-size text-decoration and color`

Please let's work on these issues and make sure we have the same design as the original template, don't worry I'm sure you can do it 😉